source/gems/dry-component/booting-a-dependency.html.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: Booting a Dependency
 layout: gem-single
+name: dry-component
```

Comment:
@timriley It was something I wondered too but could not work out a clean way to do it. Thanks for the suggestion @solnic, I will give a it a try.